### DISCUSSION TITLE
Specific value not being indexed

### DISCUSSION THREAD

**Discussion by ergonrod** [ravendb/ravendb#14212] [Q&A] [answered]
Greetings,

So I have a collection that has a Department field

![image](https://user-images.githubusercontent.com/19294010/168333500-7ae09dc7-5eef-474f-a3d6-86d22feeec29.png)

As you can see, there are a couple of documents that has the value 'IT'

For some reason the word 'IT' is not being indexed as shown below

![image](https://user-images.githubusercontent.com/19294010/168334286-0bb94b53-555b-4b9d-81d1-435c7e7a120f.png)

If I replace one of those 'IT' by 'TI', then 'TI' is indexed as normally.

![image](https://user-images.githubusercontent.com/19294010/168334509-faed35ff-f659-4282-8dfe-e91d4003ec0c.png)

What could be causing this behavior?

Thanks,
Erick

**Comment by ppekrol** [comment, marked as answer]
Hi,

the terms depends on the analyzer that is used (some of them are ignoring common words like 'it', 'and', 'the'). What kind of analyzer you are using for the Department field and what field options are set for that field?

You can read more about analyzers here: https://ravendb.net/docs/article-page/5.3/csharp/indexes/using-analyzers#understanding-analyzers

**Reply by ayende** [reply]
Slightly more context - you are likely using the `StandardAnalyzer`, which is meant to handle things like full text search.
In those cases, frequently found words( such as: it, and, the, etc) are filtered out.

For your need, you should probably use the Whitespace Analyzer, instead.

**Reply by ergonrod** [reply]
Go it!

Thank you both @ppekrol @ayende

Regards,
Erick
